src/quickgui/qgsquickutils.h

```diff
@@ -21,14 +21,17 @@
 #include <QString>
 
 #include "qgis.h"
+#include "qgsmessagelog.h"
+#include "qgsquickmapsettings.h"
 #include "qgis_quick.h"
 
+class QgsCoordinateReferenceSystem;
+
 /**
  * \ingroup quick
  *
- * Encapsulating the common utilies for QgsQuick library.
+ * Singleton encapsulating the common utilies for QgsQuick library.
```

Comment:
the version on master is better

eeeeeee!!! this file changed a lot, you copy-pasted the old version, right!?!